styleguide/source/_patterns/03-organisms/member-feature-rail.json

```diff
@@ -0,0 +1,107 @@
+{
+  "memberFeatureRail": {
+    "articleStubs": [
+      {
+        "hasVideo": "true",
+        "link": {
+          "title": "Agent of Change",
+          "href": "http://www.google.com",
+          "text": "Agent of Change"
+        },
+        "image": {
+          "alt": "alt text",
+          "src": "http://via.placeholder.com/800x600x186?l=3:2|800x600&s=36",
```

Comment:
Take make this more inline with the Zeplin mockups can we make the images
  width: 176px;
  height: 116px;